cdap-app-templates/cdap-etl/cdap-etl-proto/src/main/java/io/cdap/cdap/etl/proto/v2/ArgumentMapping.java

```diff
@@ -26,10 +26,21 @@ public class ArgumentMapping {
   private final String source;
   @Nullable
   private final String target;
+  @Nullable
+  private final TriggeringPipelineId pipelineId;
```

Comment:
rename this to just `pipeline`. It's a little more future-proof in case we ever need to add more fields to the TriggeringPipelineId object and it no longer is an id.

Reply:
updaed